src/test/java/io/epirus/console/wallet/WalletCreateCommandTest.java

```diff
@@ -26,14 +28,16 @@
 
 public class WalletCreateCommandTest {
 
+    @TempDir File tempFolder;
```

Comment:
This will cause a lot of machines to run out of swap space, as we create a lot of projects and don't necessarily delete them until we're done - /tmp will fill up and system may lock up, etc.

Reply:
ok